application/asana/v0/config/setup.json

```diff
@@ -0,0 +1,27 @@
+{
```

Comment:
Do you miss `base-url`?

Reply:
"base-url" is left only for the mock server. I will update the client.go/newClient to use default baseURL